packages/site/src/data/graphql/tools.ts

```diff
@@ -3,10 +3,234 @@ import { Tool } from "@framework/system/src/models/tool"
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Fauna",
+		author: "Fauna",
+		description: "Instantly execute GraphQL queries on your data",
+		image: "https://avatars.githubusercontent.com/u/1477000?s=200&v=4",
+		href: "https://fauna.com/graphql",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Graphene-Python",
+		author: "graphene-Python",
+		description: "GraphQL in Python Made Easy",
+		image: "https://avatars.githubusercontent.com/u/15002022?s=200&v=4",
+		href: "https://graphene-python.org/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "AWS AppSync",
+		author: "AWS",
+		description:
+			"Accelerate application development with serverless GraphQL and Pub/Sub APIs",
+		image:
+			"https://upload.wikimedia.org/wikipedia/commons/thumb/9/93/Amazon_Web_Services_Logo.svg/1024px-Amazon_Web_Services_Logo.svg.png?20170912170050",
+		href: "https://aws.amazon.com/appsync/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Apollo Client Devtools",
+		author: "Apollo",
+		description:
+			"Apollo Client Devtools is a Chrome extension for the open-source GraphQL client, Apollo Client.",
+		image:
+			"https://lh3.googleusercontent.com/T2x07f8UK9VlXmqQF-8pBfwEpVl5XJe7GzQ_GZPL0rvFpvru19rz28l65ncAeZqVsNVuNvqu6Yi-P0yMo1wnN5jYVfs=w128-h128-e365-rj-sc0x00ffffff",
+		href: "https://chrome.google.com/webstore/detail/apollo-client-devtools/jdkknkkbebbapilgoeccciglkfbmbnfm?hl=en-US",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "GraphQL Voyager",
+		author: "Ivan Goncharov",
+		description: "Represent any GraphQL API as an interactive graph",
+		image:
+			"https://pbs.twimg.com/profile_images/780770136701669376/zdsTOcpK_400x400.jpg",
+		href: "https://ivangoncharov.github.io/graphql-voyager/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "GraphQL Config",
+		author: "The Guild",
+		description:
+			"One configuration for all your GraphQL tools. The easiest way to configure your development environment with your GraphQL Schema.",
+		image:
+			"https://pbs.twimg.com/profile_images/1471121634606161920/ao7qMZBJ_400x400.jpg",
+		href: "https://www.the-guild.dev/graphql/config",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "GraphQL Inspector",
+		author: "The Guild",
+		description:
+			"Validate schemas and detect changes. Receive schema change notifications. Keep Operations and Fragments consistent.",
+		image:
+			"https://pbs.twimg.com/profile_images/1471121634606161920/ao7qMZBJ_400x400.jpg",
+		href: "https://www.the-guild.dev/graphql/inspector",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Apollo Workbench",
+		author: "Apollo",
+		description:
+			"Apollo Workbench is a VS Code extension that helps you design and reason about your organization's graph without writing any server code.",
+		image:
```

Comment:
This image appears to be broken. Please update it.

Screenshot:
![Screen Shot 2022-11-08 at 4 27 12 PM](https://user-images.githubusercontent.com/20880360/200706072-66fb80bc-c6d0-4919-9fdb-8bbf1dff04fd.png)